### PR TITLE
Fixed documentation: recommended to call the jwt endpoint

### PR DESCRIPTION
**Pull request by rfreitas**
### Scope

Users are in way being guided to create their own JWTs, which we do not way. And if they should just call the endpoint, the inner structure of the JWT is not their concern.

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://160-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 1]
```diff
@@ -328,25 +328,8 @@ You can generate your own JWTs.
...
-  The one-time use unique identifier string. Use a 64 bit random string to avoid collisions. E.g. `"JTiYyyRk3w8"`
-
-- `uuid {String} required`
-
-  A unique ID that identifies your API token in our database. This can be shared publicly and is **not** the same as your API Token. We will provide you with your uuid on request.
-
-- `ref {String} required`
-
-  The HTTP referrer of the page where the SDK is initialised. See the API [documentation](https://onfido.com/documentation#json-web-tokens) for allowed formats.
+In order to create the necessary JWT token call the onfido api endpoint on your backend.
```
Comment (hjheath) [inline README.md]: You are repeating line 320 here. We can get rid of  the `Generate your own` section and make line 316 specify only one way to generate them. We should also upgrade the expiry time.

Would you like me to make the changes?

Comment (rfreitas) [inline README.md]: yes, I missed that, thanks! I have done some changes, but feel free to adjust things.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://160-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 2]
```diff
@@ -311,25 +311,14 @@ const sendToServer = ({documentCapture, faceCapture}) => {
 
 ## Authentication
 
-Clients are authenticated using JSON Web Tokens (JWTs). The tokens are one use only and expire after 30 minutes. See [here](https://jwt.io/) for details on how JWTs work.
+Clients are authenticated using JSON Web Tokens (JWTs). The tokens expire 90 minutes after being created.
```
Comment (hjheath) [inline README.md]: 24 hours at the moment, we will decrease to 90 minutes soon, but this is currently inconsistent with the API docs.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://160-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**